src/tribol/interface/tribol.cpp

```diff
@@ -726,8 +789,8 @@ void registerRealElementField( integer meshId,
 {
    MeshManager & meshManager = MeshManager::getInstance();
 
-   SLIC_ERROR_IF(!meshManager.hasMesh(meshId), "tribol::registerRealElementField(): " << 
-               "no mesh with id " << meshId << " exists.");
+   SLIC_ERROR_ROOT_IF(!meshManager.hasMesh(meshId), "tribol::registerRealElementField(): " << 
+                      "no mesh with id " << meshId << " exists.");
 
    MeshData & mesh = meshManager.GetMeshInstance( meshId );
 
```

Comment:
This is tricky. Some Tribol API functions call this routine, but a host-code may not call those functions for null-meshes (currently), so not all ranks would get into this call, which probably poses an issue. Going to leave for now since the checks are only for null pointers.